src/dtls/ossl.rs

```diff
@@ -125,17 +137,17 @@ pub fn dtls_ssl_create(ctx: &SslContext) -> Result<Ssl, DtlsError> {
 }
 
 /// Keying material used as master key for SRTP.
-pub struct KeyingMaterial([u8; 60]);
```

Comment:
It might be possible to retain this static allocation via const generics, but it feels really a bit overkill since the keying material is generated rarely

Reply:
Let's get rid of.